ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -423,6 +423,172 @@ public final class AliasTokens {
         case brandStroke1
         case brandStroke1Pressed
         case brandStroke1Selected
+        // Dark red
+        case darkRedBackground2
+        case darkRedForeground2
```

Comment:
These colors are called out in designs as being a separate "set" of colors: Persona Colors. Shouldn't that mean they live as a separate top-level `TokenSet` within `AliasTokens`?

Reply:
Additionally, I'm not convinced that these should be alias tokens at all. Let's follow up with design and see if there's sufficient justification for these at all.

Reply:
> These colors are called out in designs as being a separate "set" of colors: Persona Colors. Shouldn't that mean they live as a separate top-level `TokenSet` within `AliasTokens`?

So far we've opted to add all new fluent 2 tokens in the same set and wanted to keep it consistent with these alias tokens. But we could move these to a separate set after discussing with design.